### PR TITLE
Update CTS csv files.

### PR DESCRIPTION
**Pull request by Jeremy-Kemp**
This removes API specific csv files, as these are no longer required.

This also updates `opencl_conformance_tests_quick.csv` to include >= 2.0 tests.

No changes were required for `opencl_conformance_tests_full.csv`.

Addresses #863

### CODE REVIEW

**Review by kpet — DISMISSED** [review]
LGTM, it's a nice step forward. A few things worth keeping in mind for future changes:
- The D3D tests aren't in any lists
- The media surface sharing tests aren't in any list
- The presence of the "quick" lists is IMHO debatable.

**Comment by gwawiork** [comment]
regarding spirv_new tests should be
`test_spirv_new --spirv-binaries-path spirv_bin`
or just leave default
`test_spirv_new`

**Comment by alycm** [comment]
> regarding spirv_new tests should be
> `test_spirv_new --spirv-binaries-path spirv_bin`
> or just leave default
> `test_spirv_new`

I was just trying out `opencl_conformance_tests_full.csv` and hit this issue too, using `test_spirv_new --spirv-binaries-path spirv_bin` fixed it for me, I imagine that no arguments is fine too.

**Comment by Jeremy-Kemp** [comment]
Thanks very much for checking it.

Did something change with that test to alter the arguments as I've not changed the csv for that test?

I don't have a hand in this one, so have no preference regarding arguments so will update to `test_spirv_new --spirv-binaries-path spirv_bin` given that you both mentioned it.

**Comment by alycm** [comment]
> Did something change with that test to alter the arguments as I've not changed the csv for that test?

Yes, `test_spirv_new` now doesn't recognize the `-ILPath` argument so running with this spreadsheet causes the test to fail with an unrecognized argument error.

It's not caused by a change in this branch, the existing CSVs were just stale.

**Review by kpet — DISMISSED** [review]

File: test_conformance/opencl_conformance_tests_quick.csv [hunk 1]
```diff
@@ -79,3 +79,29 @@ Contractions,contractions/test_contractions
...
+Non Uniform Work Groups,non_uniform_work_group/test_non_uniform_work_group
+Pipes,pipes/test_pipes
+SVM,SVM/test_svm
+Workgroups,workgroups/test_workgroups
+
+#####################################
+# OpenCL 2.1 tests
+#####################################
+Device timer,device_timer/test_device_timer
+SPIRV new,spirv_new/test_spirv_new -ILPath spirv_bin
```
Comment (gwawiork) [inline test_conformance/opencl_conformance_tests_quick.csv:98]: @Jeremy-Kemp please update this line as well.

**Review by gwawiork — APPROVED** [review]
LGTM

File: test_conformance/opencl_conformance_tests_full_no_math_or_conversions.csv [hunk 2]
```diff
@@ -39,23 +39,23 @@ Device Partitioning,device_partition/test_device_partition
...
+Images (clCopyImage max size),images/clCopyImage/test_cl_copy_images max_images
+Images (clReadWriteImage),images/clReadWriteImage/test_cl_read_write_images
+Images (clReadWriteImage pitch),images/clReadWriteImage/test_cl_read_write_images use_pitches
+Images (clReadWriteImage max size),images/clReadWriteImage/test_cl_read_write_images max_images
+Images (clFillImage),images/clFillImage/test_cl_fill_images
+Images (clFillImage pitch),images/clFillImage/test_cl_fill_images use_pitches
+Images (clFillImage max size),images/clFillImage/test_cl_fill_images max_images
+Images (Samplerless),images/samplerlessReads/test_samplerless_reads
+Images (Samplerless pitch),images/samplerlessReads/test_samplerless_reads use_pitches
+Images (Samplerless max size),images/samplerlessReads/test_samplerless_reads max_images
```
Comment (alycm) [inline test_conformance/opencl_conformance_tests_full_no_math_or_conversions.csv:58]: I can't comment on the exact line, but further down this file there is still an `-ILPath`.

Comment (alycm) [reply test_conformance/opencl_conformance_tests_full_no_math_or_conversions.csv:58]: Also the renamed `opencl_conformance_tests_full_spirv.csv ` file but I can't even comment on that file nevermind the line!

**Review by alycm — APPROVED** [review]
LGTM.

**Comment by alycm** [comment]
I'm going to merge this to get it in, I'll push the `-ILPath` fixes separately.

File: test_conformance/opencl_conformance_tests_full_no_math_or_conversions.csv (hunk 2 above)
Comment (alycm) [reply test_conformance/opencl_conformance_tests_full_no_math_or_conversions.csv:58]: Done here: https://github.com/KhronosGroup/OpenCL-CTS/pull/981

**Pull request merged**